docs/user/concepts.md

```diff
@@ -159,6 +160,22 @@ To further simplify device registration, Provisioning Tokens can be created to a
 devices to automatically connect to a team without having to manually register them first.
 The token can also be configured to assign a device directly to a Node-RED instance within the team.
 
+### Device Actions
+
+**Introduced in FlowFuse 2.2**
+
+Devices have an action dropdown menu that can request the device agent to suspend, start and restart the
+Node-RED instance running on the device.
+
+This is useful when you want to temporarily stop Node-RED, restart it to pick up updates or to aid diagnosing issues.
+
+While a device is suspended, the Device Agent will continue to run and report device status
+to the platform, but your Node-RED flows will not stopped. If the Device Agent is restarted,
+Node-RED will be continue to be suspended. In order to exit suspended mode, start action must be requested.
+
+NOTE: Device Actions require FlowFuse to be configured with a broker. 
+Please contact your FlowFuse administrator for more information or see the [broker configuration](../install/configuration.md#mqtt-broker-configuration) documentation.
+
```

Comment:
```suggestion
```

remove docs, feature is self explaining - for later follow up task

Reply:
Applying this patch - overreaching issue addressed in https://github.com/FlowFuse/flowfuse/issues/3566